components/places/android/src/main/java/mozilla/appservices/places/LibPlacesFFI.kt

```diff
@@ -161,6 +161,15 @@ internal interface LibPlacesFFI : Library {
         error: RustError.ByReference
     ): RustBuffer.ByValue
 
+    fun places_get_visit_page_with_bound(
+        handle: PlacesConnectionHandle,
+        offset: Long,
+        bound: Long,
```

Comment:
nit: I think it would be better to switch the order of these parameters, e.g. pass in `bound, offset` rather than `offset, bound`. It will help the caller to form a more accurate mental model of how the underlying query works; "first skip directly to `bound`, then skip over `offset`  items from there".

Reply:
Done.